codes/1916_cpp_dbstn322.cpp

```diff
@@ -0,0 +1,45 @@
+#include <cstdio>
+#include <vector>
+#include <utility>
+#include <queue>
+
+using namespace std;
+
+vector<pair<int, int>> edges[100001];
+priority_queue<pair<int, int>> shortDist;
```

Comment:
<,>를 2번 이상 연속으로 쓰면 컴파일러가 비트연산으로 잘못처리해서 컴파일 에러가 날 수 있습니다.
```suggestion
vector<pair<int, int> > edges[100001];
priority_queue<pair<int, int> > shortDist;
```